components/cam/cime_config/config_component.xml

```diff
@@ -105,6 +105,9 @@
       <value compset="_CAM5%SP2V1"           >-SPCAM_microp_scheme m2005 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero </value>
       <value compset="_CAM5%SP[12]V1_"       >-crm_nx 64 -crm_ny 1 -crm_nx_rad 16 -crm_ny_rad 1 </value>
       <value compset="_CAM5%SP[12]V1-TEST_"  >-crm_nx  8 -crm_ny 1 -crm_nx_rad  1 -crm_ny_rad 1 </value>
+      <value compset="_CAM5%SP1V1-MAML"      >-use_MAML -crm_ny 1 -crm_ny_rad 1 </value>
+      <value compset="_CAM5%SP1V1-MAML_"     >-crm_nx 64 -crm_nx_rad 64 </value>
```

Comment:
Should we just combine these lines?

Reply:
I had it combined originally and only found out that the final CAM_CONFIG_OPT had both -crm_nx_rad 16 and -crm_nx_rad 64, so I had to break it up.